controllers/data.js

```diff
@@ -4,11 +4,14 @@ const _ = require('lodash');
 const Constants = require('./constants');
 const Request = require('./request');
 
+const APP_TOKEN = 'CqcTvF7wVsI8IYAq7CdZszLbU';
+
 class Data {
 
```

Comment:
Could replace the next function with 

```
const url = Request.buildURL(Constants.DATA_AVAILABILITY_URL, {
    app_token: APP_TOKEN,
    entity_id: regions.map(_.property('id')).join(',')
});

return Request.getJSON(url);
```

Since `Request.getJSON(url)` is a promise.